pkg/distributor/distributor.go

```diff
@@ -98,8 +107,13 @@ type Distributor struct {
 // create a Distributor
 type Config struct {
 	EnableBilling bool
+	HAReplicas    bool
+	HARejectOne   bool
+	KVStore       string
 	BillingConfig billing.Config
 	PoolConfig    ingester_client.PoolConfig
+	TrackerConfig HATrackerConfig
+	ConsulConfig  ring.ConsulConfig
```

Comment:
`ConsulConfig` (and eventually `KVConfig`, see below) should be part of `TrackerConfig` IMHO.

Reply:
Looks like there is already a ConsulConfig in `TrackerConfig`.